koku/masu/processor/ocp/ocp_cost_model_cost_updater.py

```diff
@@ -54,8 +54,8 @@ def __init__(self, schema, provider):
             )
             self._distribution_info = cost_model_accessor.distribution_info
 
-    def _build_node_tag_cost_case_statements(
-        self, rate_dict, start_date, default_rate_dict={}, unallocated=False, node_core=False
+    def _build_node_tag_cost_case_statements(  # noqa: C901
```

Comment:
Truthfully I hate all these tag case statement builders! It's impossible to know what happens here without printing out the sql afterwards. We should find a better solution to this, maybe structured SQL files would be better. Perhaps we should include that work into the dataclass refactor issue and just refactor cost model stuff in general.